### DISCUSSION TITLE
Getting `textarea` value only returns first 9 lines?

### DISCUSSION THREAD

**Discussion by jgarrow** [cypress-io/cypress#15795] [Questions and Help]
I need to run an assertion on a `textarea`'s value, but Cypress is only ever getting the first 9 lines. I've tried the following approaches and get the same results with each  :

```js
cy.get('textarea').should('have.value', LONG_EXPECTED_VALUE)
```

```js
cy.get('textarea')
      .invoke('val')
      .then(actualValue => {
        expect(actualValue).to.equal(LONG_EXPECTED_VALUE);
      });
```

```js
// LONG_EXPECTED_VALUE has been formatted here for readability in lieu of lots of `\n` characters
const LONG_EXPECTED_VALUE = 'const typeDefs = gql`
  type SpaceCat {
    id: ID!
    name: String!
    age: Int
    missions: [Mission]
  }
`
/* your code:
const typeDefs = gql`
  # write your types here
`
*/'
```

I have tried numerous combinations and structures of `LONG_EXPECTED_VALUE`. At first I thought it was because the bottom portion is a comment, but removing the comment and having it be a standalone variable declaration made no difference. Then I thought it might be an issue with the back ticks with `gql`, but removing those didn't change anything either. I noticed after reordering `LONG_EXPECTED_VALUE` to look like the following, it still cuts it off and after counting the lines, I realized in all instances, Cypress was only ever returning the first 9 lines:

```js
// LONG_EXPECTED_VALUE has been formatted here for readability in lieu of lots of `\n` characters
const LONG_EXPECTED_VALUE = '/* your code:
const typeDefs = gql`
  # write your types here
`
*/

const typeDefs = gql`
  type SpaceCat {
    id: ID!
    name: String!
    age: Int
    missions: [Mission]
  }
`'
```

Is this behavior to only return the first 9 lines intentional? If so, is there a way to specify that I want the full value?

**Comment by bahmutov** [comment]
Can you set up a failing test example in https://github.com/cypress-io/cypress-test-tiny or any other public repo please for us to see the error? Or put the debugger into the `.then()` callback to print the value? and add screenshots of the values?

```js
.then(actualValue => {
     console.log(actualValue)
     debugger
     expect(actualValue).to.equal(LONG_EXPECTED_VALUE);
 });
```

**Reply by jgarrow** [reply]
It's a private repo, so I can't share much of the code or endpoints. But here are some screenshots after logging out `actualValue` and throwing in a debugger. For reference, we use a module called `outdent` that takes a string and preserves any whitespace (like new lines and tabs) so the code is more readable and we don't have to use `\n` and `\t` everywhere.

What's seen on the actual webpage of what's actually in the `textarea`:
<img width="620" alt="Screen Shot 2021-04-05 at 3 50 30 PM" src="https://user-images.githubusercontent.com/32886852/113635037-1f6f1380-962d-11eb-904f-2e86cdd5294b.png">

What gets logged out from `console.log(actualValue)` just before the debugger:
<img width="604" alt="Screen Shot 2021-04-05 at 3 52 48 PM" src="https://user-images.githubusercontent.com/32886852/113635316-a4f2c380-962d-11eb-8561-56514746b834.png">

A full view of the Cypress log, the webpage, and console before continuing on with the debugger
<img width="1237" alt="Screen Shot 2021-04-05 at 3 52 39 PM" src="https://user-images.githubusercontent.com/32886852/113635392-c3f15580-962d-11eb-9705-1fe396482447.png">

The error in Cypress
<img width="641" alt="Screen Shot 2021-04-05 at 4 45 36 PM" src="https://user-images.githubusercontent.com/32886852/113635713-5abe1200-962e-11eb-827c-a6da488f556e.png">
